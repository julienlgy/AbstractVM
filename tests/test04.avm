push float(2.0)
push float(3.0)
div
assert float(0.6666667)
push double(2.0)
push double(3.0)
div
assert double(0.666666666666667)
push float(50.0)
push double(2000.0)
div
assert double(0.025)
dump
exit;