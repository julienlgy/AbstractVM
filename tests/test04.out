0.025
0.666666666666667
0.6666667
